core/mm/vm.c

```diff
@@ -566,9 +571,14 @@ TEE_Result vm_remap(struct user_mode_ctx *uctx, vaddr_t *new_va, vaddr_t old_va,
 	struct vm_region *r_first = NULL;
 	struct fobj *fobj = NULL;
 	vaddr_t next_va = 0;
+	vaddr_t _new_va = 0;
```

Comment:
Please use a proper name, the underscore prefix is reserved for other situations.